src/db.rs

```diff
@@ -291,80 +278,197 @@ pub fn subgraph_deployment_change(
                 curation_pool.ordinal,
                 Operation::Update, // Update will create the entity if it does not exist
             )
-            .change(
-                "signalledTokens",
-                BigInt::from_str(&curation_pool.new_signal).unwrap(),
-            );
+            .change("signalledTokens", &curation_pool.new_signal);
     }
-    for indexing_reward in indexing_rewards.indexing_rewards {
+
+    // just realized I need to add a store to aggregate indexer rewards per subgraph deployment since they are for all indexers
+    // this code needs to be updated. db should recive some stores, not indexing rewards directly. allocations should use indexing rewards directly
```

Comment:
@sahra-karakoc this is the comment we discussed through Slack, if it's solved it would be a good idea to remove this so we don't get confused in the future 😅

Reply:
Deleted, thanks!